### PR TITLE
ENH: Adding materials for PNNL atsci course

### PR DESCRIPTION
**Pull request by AdamTheisen**
@zssherman just a heads up

### CODE REVIEW

File: Tutorials/2024_PNNL_Atmospheric_Course/6-ACT-TRACER-Dust.ipynb [hunk 1]
```diff
@@ -0,0 +1,731 @@
+{
```
Comment (zssherman) [inline Tutorials/2024_PNNL_Atmospheric_Course/6-ACT-TRACER-Dust.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #4.    display.plot('signal_return_cross_pol', cb_friendly=True, vmin=-20, vmax=20)
</pre><p>These calls should be cvd_friendly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ARM-Development/ARM-Notebooks/pull/102/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

**Comment by zssherman** [comment]
@AdamTheisen Did you want me to work on the ACT basics notebook then?

**Comment by AdamTheisen** [comment]
@zssherman yes, please do so.  I wasn't sure how far you were and didn't want to cause any issues.  I also think the work your doing on the DL wind barb overplots is great to include in one of these as well.

**Comment by zssherman** [comment]
@AdamTheisen Sounds good! Did you want me to add the DL case to basics or this notebook here?

**Comment by AdamTheisen** [comment]
@zssherman let's do it in this one here. Maybe pull data for the July 17 2022 case and see what it gives us

**Comment by zssherman** [comment]
@AdamTheisen I added the doppler lidar code. Let me know if there is any more info or anything to add to it.

**Comment by AdamTheisen** [comment]
Thanks @zssherman !  I'll merge this in and we can keep working

**Pull request merged**